subsys/nrf_security/Kconfig.psa

```diff
@@ -70,6 +70,24 @@ config MBEDTLS_PSA_KEY_SLOT_COUNT
 	  transparent keys that we can have open concurrently.
 	  The default value comes from MBedTLS.
 
+config MBEDTLS_PSA_STATIC_KEY_SLOTS
+	bool "Enable static key slots for PSA"
```

Comment:
this already exists in zephyr so should not be added again

Reply:
It exists indeed but it has the dependency MBEDTLS_BUILTIN which is not compatible with nrf_security so I cannot remove it from here :/